source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -186,7 +186,7 @@ - (void)update:(NSString *) jsonStr
         
         NSDictionary *viewMap = NSDictionaryOfVariableBindings(view, scrollview);
         NSArray<NSString *> *formats =
-        [NSArray arrayWithObjects:@"H:|-10-[view(<=scrollview)]-10-|", @"V:|-[view(>=scrollview)]-|",nil];
+        [NSArray arrayWithObjects:@"H:|-10-[view(<=scrollview)]-10-|", @"V:|-10-[view(<=scrollview)]-10-|",nil];
```

Comment:
What does this fix?

Reply:
my sample app vertical scroll didn't work

---
In reply to: [155118721](https://github.com/Microsoft/AdaptiveCards/pull/1029#discussion_r155118721) [](ancestors = 155118721)